why_foss/README.md

```diff
@@ -0,0 +1,9 @@
+# FOSS
```

Comment:
This ReadMe should be changed to `Why should I contribute to Open Source` and relevant blog post link can be added to the description.

Reply:
I updated the ReadMe file and also added the link to your blog.